server/models/Player.js

```diff
@@ -0,0 +1,19 @@
+//Player
+
+
+var mongoose = require('mongoose')
+  , ObjectId = mongoose.SchemaTypes.ObjectId
+  ;
+// define player schema
+var productSchema = mongoose.Schema({
+  created:                  { type: Date, default: Date.now }
+  , updated:                { type: Date, default: Date.now }
+  , firstname:              { type: String, required: '{PATH} is required!' }
+  , lastname:               { type: String, required: '{PATH} is required!' }
+  , playernumber:           { type: String }
+  , team:                   { type: ObjectId, ref: 'Team' }
+  , playerimage:              { type: String }
```

Comment:
You don't need to change anything here now, but worth noting for the future: Use camelCase for variable names. In this case, you should have called these fields playerNumber and playerImage (capitalize new words in the string). This makes for easier reading of the variables.